### DISCUSSION TITLE
EBYTE E79-900DM2005S / CC1352P pinout ?

### DISCUSSION THREAD

**Discussion by capacom** [Koenkk/zigbee2mqtt#10546] [Devices]
Hello,

I have been browsing around without success to understand which DIO should be used for TX and RX with CC1352P. 

I found some stories about DIO2 and DIO3 but my board start with DIO_7 ?

Thanks

**Comment by 1igeza** [comment]
It should be DIO_12/DIO_13, 
![cc1352_e79_sch](https://user-images.githubusercontent.com/39980292/148095175-28530ce8-88e5-4ede-af52-c6f5aa2ebdb7.png)

**Reply by capacom** [reply]
Thanks for this reply.

I do confirm I found it is DIO_13 / DIO_12 while reading the .patch file of the coordinator.

However so far it is not working with Zigbee2MQTT. I will cross check.

I found a thread talking about the power of the module that has to be reduce to 5 with a custom compile of the coordinator in order to have this module working... I will give a try too later.

**Reply by 1igeza** [reply]
I was using E79 (I gave it to my brother and it works perfectly for him) I was using firmware from egony and from Koenkk and both worked correctly. First time I had to flash it with j-link but after first flashing I was regular updating it by USB to TTL (UART). I have some firmware files which I was using if you want you can try them.
[CC1352P_2_20200830.zip](https://github.com/Koenkk/zigbee2mqtt/files/7809349/CC1352P_2_20200830.zip)

**Reply by capacom** [reply]
Just did a try with your version and got a working condition.

Thanks a lot. 

I will have to cross check what is wrong with the latest version.

**Reply by capacom** [reply]
To complete this thread it's working fine with latest CC1352P2_CC2652P_launchpad_coordinator_20211217.zip

I was using the CC1352P2_CC2652P_other_*.zip because the tested adapter brought me wrong with : 

Electrolama zoe2 | CC1352P(Ebyte E79) | CC1352P2_CC2652P_other_*.zip
